sapp/ui/server.py

```diff
@@ -99,5 +99,12 @@ def start_server(
     if static_resources:
         application.static_folder = static_resources
     if debug:
-        CORS(application, resources={r"/graphql": {"origins": "http://localhost:3000"}})
+        default_port = os.environ.get("PORT") or 3000
+        host = os.environ.get("HOST") or "localhost"
+        CORS(
+            application,
+            resources={
+                r"/graphql": dict(origins="http://{hostname}:{port}".format(hostname=host, port=default_port)
```

Comment:
```suggestion
                r"/graphql": dict(origins="http://{hostname}:{port}".format(hostname=host, port=default_port))
```

Looks like we are missing a closing parentheses here

https://github.com/facebook/sapp/runs/4885612036?check_suite_focus=true